scripts/library.sh

```diff
@@ -696,7 +696,7 @@ function get_latest_knative_yaml_source() {
     local major_minor="${branch_name##release-}"
     # Find the latest release manifest with the same major&minor version.
     local yaml_source_path="$(
-      gsutil ls gs://knative-releases/${repo_name}/previous/v${major_minor}.*/${yaml_name}.yaml 2> /dev/null \
+      gsutil ls gs://knative-releases/"${repo_name}"/previous/v"${major_minor}".*/"${yaml_name}".yaml 2> /dev/null \
```

Comment:
This would be better with double quotes across most of the line, i.e. before `gs:` and after `.yaml` rather than all the groupings inside

Reply:
done